modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase

```diff
@@ -2439,6 +2439,55 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-10066. User can only view the allowed quantities that match the minimum, max and multiple quantities of a product"
+	@priority = 5
+	test CanViweOnlyCorrectAllowedQuantitiesAreSelectableInProductDetails {
```

Comment:
CanViewOnlyCorrectAllowedQuantitiesAreSelectableForProduct

Please combine these 2 tests and make an assertion first on the product card, then on the product details page. It's a repetition and can be worked under one test. Thanks!